www/js/incident/post-trip-prompt.js

```diff
@@ -118,21 +118,8 @@ angular.module('emission.incident.posttrip.prompt', ['emission.plugin.logger'])
 
   var displayCompletedTrip = function(notification, state, data) {
       $rootScope.displayingIncident = true;
+      $rootScope.notificationData = notification.data;
       Logger.log("About to display completed trip from Notification");
-      $state.go("root.main.enketosurvey", {
```

Comment:
I don't think you need this when you already have the code in `www/js/tripconfirm/post-trip-prompt.js`